resources/views/account/public-profile/index.blade.php

```diff
@@ -2,43 +2,38 @@
 
 @section('content')
 
-    <div class="container">
-        <ol class="breadcrumb">
-            <li><a href="/">Home</a></li>
-            <li class="active"><a href="{{ route('speakers-public.index') }}">Speakers</a></li>
-        </ol>
+    <div class="container body">
+        <div class="col-md-10 col-md-push-1">
+            <h1>Speaker Profiles</h1>
```

Comment:
Was this breadcrumb bad or is this a merge wonky-ness?

Reply:
This looks like a merge issue. I'll check it out.